### PR TITLE
Distro image creation now supports additional packages

### PR DESCRIPTION
**Pull request by dlsloan**
Added option to add additional packages to distro image creation script
on command line.  This helps with image-setup when automating testing
of kernel images using qemu.

### CODE REVIEW

File: scripts/create [hunk 1]
```diff
@@ -47,11 +48,11 @@ trap cleanup EXIT
 
 if [ $DISTRO == "debian" ]; then
     debootstrap \
-	--include=less,locales-all,vim,sudo,openssh-server,jed,pciutils,bash-completion,psmisc,htop,pydf\
+	--include=less,locales-all,vim,sudo,openssh-server,jed,pciutils,bash-completion,psmisc,htop,pydf,${PKGS}\
```
Comment (lsgunth) [inline scripts/create]: If PKGS is not set does it still work with the trailing comma? If so this is fine.

Comment (dlsloan) [reply scripts/create]: Yup, tested that one

**Comment by sbates130272** [comment]
Applied thanks @dlsloan!

**Comment by sbates130272** [comment]
Hmm for some reason there was no merge so I am reopening.

**Comment by dlsloan** [comment]
Messed up this branch, resubmitted clean

**Pull request closed without merging**